app/assets/javascripts/discourse/tests/acceptance/dashboard-test.js

```diff
@@ -140,8 +140,10 @@ acceptance("Dashboard", function (needs) {
   test("new features", async function (assert) {
     await visit("/admin");
 
+    await click(".dashboard .navigation-item.new-features .navigation-link");
+
     assert.ok(exists(".dashboard-new-features"));
-    assert.ok(exists(".dashboard-new-features .new-features-release-notes"));
+    assert.ok(exists("img.screenshot"));
```

Comment:
IMO this is too vague, should be like `.new-feature-item .new-feature-content img.screenshot` (but with BEM updates )